kod/object/item/passitem/numbitem/food/soup.kod

```diff
@@ -49,6 +49,7 @@ properties:
    viFilling = 20
    viNutrition = 9
    piNumber = 1
+   piItem_flags = 82   % Slight blue hue to set it apart from bowl of stew
```

Comment:
Please use the symbolic names from blakston.khd rather than hard-coding the values.